baseplate/integration/wrapped_context.py

```diff
@@ -0,0 +1,22 @@
+import logging
+
+
+class WrappedRequestContext(object):
+    def __init__(self, context, trace=None):
+        self.__dict__['_context'] = context
+        self.__dict__['logger'] = logging.getLogger(self.__class__.__name__)
+
+    def __getattr__(self, attr):
+        return getattr(self._context, attr)
+
+    def __setattr__(self, attr, value):
+        if attr in self._context.__dict__:
+            self.logger.debug("Assigning attr=%s of wrapped request context.")
```

Comment:
I think the case where we're logging for changing the wrapped request context will definitely be spammy (since we assign a ton of context objects when a new wrapped context gets created for local spans), removing that one. The reason I added the one for setting attributes of the underlying context is to provide explicit messaging in case developers are inadvertently messing with the context itself when they want to manipulate WrappedContext attrs and trying to debug whether or not that's the case.